<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesWindow</class>
 <widget class="QDialog" name="NotesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="NotesFunctionsBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>560</width>
     <height>440</height>
    </rect>
   </property>
   <property name="title">
    <string>Notes Functions</string>
   </property>
   <widget class="QGroupBox" name="CreateNoteBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Create Note</string>
    </property>
    <widget class="QPushButton" name="CreateNoteButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Note</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="UpdateNoteBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Update Note</string>
    </property>
    <widget class="QPushButton" name="UpdateNoteButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Note</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="DeleteNoteBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Delete Note</string>
    </property>
    <widget class="QPushButton" name="DeleteNoteButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Note</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TasksBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>390</width>
      <height>170</height>
     </rect>
    </property>
    <property name="title">
     <string>Tasks</string>
    </property>
    <widget class="QTableView" name="TasksTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>370</width>
       <height>140</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="NotesForTheTaskBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>390</width>
      <height>170</height>
     </rect>
    </property>
    <property name="title">
     <string>Notes For The Tasks</string>
    </property>
    <widget class="QTableView" name="NotesForTheTaskTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>370</width>
       <height>140</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>520</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout">
     <item>
      <spacer name="HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
